src/lib/src/tooltip/tooltip.component.html

```diff
@@ -2,6 +2,7 @@
   class="c-tooltip qa-tooltip"
   [matTooltip]="tooltipValue"
   [matTooltipPosition]="tooltipPosition"
+  [ngClass]="{'qa-tooltip-dotted-underline': hasUnderline}"
```

Comment:
Classes that begin with `qa-` are only for allowing QA to query our DOM elements. We have a separation between classes we use and classes QA uses so that they don't have to update their tests each time we refactor.

Since this is a modifier class we should use the BEM modifier naming format to extend the base `.c-tooltip` class: `.c-tooltip--underlined`